web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknadsbehandling/SøknadsbehandlingRoutes.kt

```diff
@@ -496,3 +486,13 @@ internal fun Route.søknadsbehandlingRoutes(
         }
     }
 }
+
+internal fun SøknadsbehandlingService.FeilVedValideringAvBehandlingsinformasjon.tilResultat(): Resultat {
+    return when (this) {
+        SøknadsbehandlingService.FeilVedValideringAvBehandlingsinformasjon.DepositumIkkeMindreEnnInnskudd -> depositumIkkeMindreEnnInnskudd
+        SøknadsbehandlingService.FeilVedValideringAvBehandlingsinformasjon.BosituasjonSamsvarerIkkeMedInformasjonIFormue -> BadRequest.errorJson(
+            "Bosituasjon samsvarer ikke med informasjon i formue",
```

Comment:
Ble ikke hele denne error-chainen litt for generisk? Det gjelder vel fremdeles kun EPS sin formue?

Reply:
ikke helt. sjekken om depositum gjøres for både søker og EPS. Der vi sjekker om bosituasjon er konsistent med formue er kun EPS.